### PR TITLE
Bauti/nft 1267 update the renft sdk to track the

### PR DESCRIPTION
**Pull request by bauti-defi**
Add `startBlock` prop to all contract deployments. The appropriate info has been added to each existing deployment.

### CODE REVIEW

File: src/deployments.ts [hunk 1]
```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
...
 
 export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V1 = {
   contractAddress: '0x4e52b73aa28b7ff84d88ea3a90c0668f46043450',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V1,
+  startBlock: 36825974,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_FUJI_TESTNET_V0 = {
```
Comment (cawfree) [inline src/deployments.ts:58]: This is strange, this address looks to be an ERC20: https://testnet.snowtrace.io/address/0x42816FA3cB0aDc3fcAdED3109323c0Bc19215084#code

Can we fix this regression?

File: src/deployments.ts [hunk 2]
```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
...
   version: WhoopiVersion.V0,
+  startBlock: 11500156,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_MAINNET_V0 = {
   contractAddress: '0x6Ee495ecEd3A0255057667FF2685e53f54A19A65',
   network: NETWORK_AVALANCHE_MAINNET,
   contractType: RenftContractType.WHOOPI,
   version: WhoopiVersion.V0,
+  startBlock: 19408332,
```
Comment (cawfree) [inline src/deployments.ts:71]: Can you share where these values come from? I can't find these values:
https://snowtrace.io/txs?a=0x6Ee495ecEd3A0255057667FF2685e53f54A19A65&p=2000

File: src/deployments.ts [hunk 3]
```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
...
   version: ResolverVersion.V0,
+  startBlock: 11500156,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_AVALANCHE_MAINNET_V0 = {
   contractAddress: '0xEBFd584AAC21dfEFF02c3d4f308B0962610a028A',
   network: NETWORK_AVALANCHE_MAINNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 19408332,
```
Comment (cawfree) [inline src/deployments.ts:111]: Struggling to find these values: https://snowtrace.io/address/0xEBFd584AAC21dfEFF02c3d4f308B0962610a028A

File: src/deployments.ts [hunk 4]
```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
...
   version: ResolverVersion.V1,
+  startBlock: 36825213,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_AVALANCHE_FUJI_TESTNET_V0 = {
   contractAddress: '0x23F7F8B03BAF01D5124255fE240E81BbBd3AEc0D',
   network: NETWORK_AVALANCHE_FUJI_TESTNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 11500156,
```
Comment (cawfree) [inline src/deployments.ts:103]: Unable to find:d https://testnet.snowtrace.io/address/0x23F7F8B03BAF01D5124255fE240E81BbBd3AEc0D

File: src/deployments.ts [hunk 5]
```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
...
 export const DEPLOYMENT_RESOLVER_POLYGON_MAINNET_V0 = {
   contractAddress: '0x6884d88Ce56C5C93F46eE23684eBA8628c90B518',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.RESOLVER,
   version: ResolverVersion.V0,
+  startBlock: 28399112,
 } as const;
 
 export const DEPLOYMENT_RESOLVER_POLYGON_MAINNET_V1 = {
   contractAddress: '0x3ddc85bb768a11b0125f4ee71cfea54e54653366',
```
Comment (cawfree) [inline src/deployments.ts:91]: Unable to find: https://testnet.snowtrace.io/address/0x3ddc85bb768a11b0125f4ee71cfea54e54653366

File: src/deployments.ts (hunk 1 above)
Comment (bauti-defi) [reply src/deployments.ts:58]: That is a polygon mainnet deployment, not AVAX

File: src/deployments.ts (hunk 2 above)
Comment (bauti-defi) [reply src/deployments.ts:71]: The contract was created on this transaction: https://snowtrace.io/tx/0x5c913c9d83b72784199b26e2c46c1cad184960df1c1f74754d747fc32b9aa129

File: src/deployments.ts (hunk 3 above)
Comment (bauti-defi) [reply src/deployments.ts:111]: contract creation txn: https://snowtrace.io/tx/0x5c913c9d83b72784199b26e2c46c1cad184960df1c1f74754d747fc32b9aa129

File: src/deployments.ts (hunk 5 above)
Comment (bauti-defi) [reply src/deployments.ts:91]: this is a polygon deployment, your link is AVAX

**Review by cawfree — APPROVED** [review]
Great work! I find it super weird that Snowtrace doesn't render the creation transaction when the rest of the block explorer is consistent.

Thanks @bauti-defi.

**Pull request merged**